src/index.html

```diff
@@ -20,8 +21,530 @@
       href="styles/index.scss"
     />
   </head>
+  <!-- #endregion -->
 
-  <body>
-    <h1>Catalog</h1>
+  <body class="page">
+    <header class="page__header header">
+      <a
+        href="#logo"
+        class="header__logo"
+      >
+        <img
+          src="images/logo.png"
+          alt="header_logo_image"
+          class="header__logo-image"
+        />
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-link">
+            <a
+              href="#apple"
+              class="header__nav-link-text is-active"
+            >
+              apple
+            </a>
+          </li>
+
+          <li class="header__nav-link">
+            <a
+              href="#samsung"
+              class="header__nav-link-text"
+            >
+              samsung
+            </a>
+          </li>
+
+          <li class="header__nav-link">
+            <a
+              href="#smartphones"
+              class="header__nav-link-text"
+            >
+              smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-link">
+            <a
+              href="#laptops_%26_computers"
```

Comment:
The href attribute contains an encoded character '%26' which represents '&'. It's better to use '&' directly in HTML unless there is a specific reason to encode it.